src/state/StateOutput.cpp

```diff
@@ -518,6 +518,154 @@ namespace polyfem
 		return fs::weakly_canonical(fs::path(output_dir) / path).string();
 	}
 
+    void State::extract_boundary_mesh_pressure()
```

Comment:
is this the same as the other one? can it be refactored into 1 function with different params?